spec/invoice_details_request/tax_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+require 'byebug'
+
+module CXML
+  module InvoiceDetailRequest
+    describe Tax do
+      it 'builds the tax partial of the cxml invoice' do 
+        builder = CXML.builder
+        data = { amount: 405, currency: 'GBP', description: "total tax", purpose: "tax",
+          category: "vat", percentage_rate: "20", location: 'GB' }
+        tax_object = described_class.new(data)
+        expect((tax_object.render(builder)).to_xml). to eq(<<~EOF
```

Comment:
- Can we put a blank between the setup and the expectation of the test?
- There's an extra set of ()